spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+describe Api::V1::RentsController do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'when fetching all the rents' do
+      let!(:rents) { create_list(:rent, 3) }
+
+      before do
+        get :index
+      end
+
+      it 'responds with the rents json' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          rents, each_serializer: RentSerializer
+        ).to_json
+        expect(response.body.to_json) =~ JSON.parse(expected)
+      end
+
+      it 'responds with 200 status' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when receiving book_id filter' do
+      let(:rents_book) { create_list(:rent, 3, book: create(:book, id: 1)) }
+      let(:rents) { create_list(:rent, 3) }
+
+      before do
+        get :index, params: { book_id: 1 }
+      end
+
+      it 'responds with the rents json' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          rents, each_serializer: RentSerializer
```

Comment:
I'm pretty sure this should be `rents_book`.
Why is this working though? It would be interesting to debug it